### PR TITLE
Bindgen patches

### PR DESCRIPTION
**Pull request by PaulHax**
- Support unsigned integer as pipeline parameter.  
- Only generate bindgen for specified language.
- Default CDN url uses package name.

### CODE REVIEW

File: src/bindgen/typescript-resources/pipelines-base-url.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import { getPipelinesBaseUrl as itkWasmGetPipelinesBaseUrl } from 'itk-wasm'
 import packageJson from '../package.json'
 
 let pipelinesBaseUrl: string | URL | undefined
-let defaultPipelinesBaseUrl: string | URL = `https://cdn.jsdelivr.net/npm/@itk-wasm/compress-stringify@${packageJson.version}/dist/pipelines`
+let defaultPipelinesBaseUrl: string | URL = `https://cdn.jsdelivr.net/npm/<bindgenPackageName>@${packageJson.version}/dist/pipelines`
```
Comment (thewtex) [inline src/bindgen/typescript-resources/pipelines-base-url.ts:5]: :eagle: :eye:

File: src/bindgen/typescript.js [hunk 2]
```diff
@@ -23,6 +23,7 @@ const interfaceJsonTypeToTypeScriptType = new Map([
   ['BOOL', 'boolean'],
   ['TEXT', 'string'],
   ['INT', 'number'],
+  ['UINT', 'number'],
```
Comment (thewtex) [inline src/bindgen/typescript.js:26]: :+1:

**Review by thewtex — APPROVED** [review]
@PaulHax thank you!!

File: src/bindgen/typescript.js (hunk 2 above)
Comment (PaulHax) [reply src/bindgen/typescript.js:26]: ✅

**Pull request merged**